src/js/edu-benefits/5490/config/form.js

```diff
@@ -537,6 +537,31 @@ const formConfig = {
             educationProgram: {
               name: {
                 'ui:title': 'Name of school, university, or training facility you want to attend'
+              },
+              educationType: {
+                'ui:options': {
+                  updateSchema: (pageData, form, schema) => {
```

Comment:
`updateSchema` is a callback that's used by the reducer: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/common/schemaform/helpers.js#L513

We only use it here and maybe one other place. What I've done below is a bit of a hack (stealing the keys from the UI object to get the original labels list). @cvalarida has  a more elegant solution in https://github.com/department-of-veterans-affairs/vets-website/pull/5157, but I thought we should discuss before making this change (especially just before launch). @cvalarida feel free to chime in here if you feel strongly we should go with your reducer changes instead!